.github/scripts/pkg-version.sh

```diff
@@ -32,4 +32,15 @@ elif [ $1 == 'core-template' ]; then
     echo "need to bump up templates' fallback version in fx-core"
     node ../.github/scripts/fxcore-sync-up-version.js
     git add ../packages/fx-core
+elif [ $1 == 'template-adaptive-card' ]; then
+    if [[ $SkipSyncup == *"template"* ]]; then
+        echo "skip sync up templates version with adaptive-card version"
+    elif [[ -z "$(git diff -- ../../templates)" ]]; then
+    echo "need bump up templates version since templates don not bump up by self"
```

Comment:
Nit: better to add indents to improve readability.

Reply:
thanks!
Updated.